<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionLayout</class>
 <widget class="QMainWindow" name="ConnectionLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ipaddress_input">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ipaddress_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ip Address</string>
    </property>
   </widget>
   <widget class="QComboBox" name="connection_type_selector">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RFC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LTE</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionTest1"/>
    <addaction name="actionTes2"/>
   </widget>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionTest1">
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
  <action name="actionTes2">
   <property name="text">
    <string>Tes2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
